psydac/api/discretization.py

```diff
@@ -204,7 +206,116 @@ def solve(self, **kwargs):
 
         return driver_solve(self.linear_system, **settings)
 
+#==============================================================================
+class DiscreteDerham(BasicDiscrete):
+    def __init__(self, *spaces):
+    
+        dim       = len(spaces) - 1
+        self._V0  = None
+        self._V1  = None
+        self._V2  = None
+        self._V3  = None
+        self._dim = dim
+        
+        if dim == 1:
+            self._V0 = spaces[0]
+            self._V1 = spaces[1]
+        elif dim == 2:
+            self._V0 = spaces[0]
+            self._V1 = spaces[1]
+            self._V2 = spaces[2]
+        elif dim == 3:
+            self._V0 = spaces[0]
+            self._V1 = spaces[1]
+            self._V2 = spaces[2]
+            self._V3 = spaces[3]
+
+    # ...
+    @property
+    def dim(self):
+        return self._dim
+
+    # ...  
+    @property
+    def V0(self):
+        return self._V0
+
+    @property
+    def V1(self):
+        return self._V1        
+
+    @property
+    def V2(self):
+        return self._V2
+        
+    @property
+    def V3(self):
+        return self._V3  
+
+#==============================================================================           
+def discretize_derham(V, domain_h, *args, **kwargs):
```

Comment:
Can we add a docstring to this important function? Also, I find that naming a de Rham complex as `V` is a little confusing, as we usually use this name for a function space.

Reply:
the name is changed V here is called Complex